### ISSUE TITLE
Ensure we support modeless dialogs/modals

### ISSUE THREAD

**Issue by cal-smith** [IBM/carbon-components-angular#999]
Reference: https://www.carbondesignsystem.com/patterns/dialog-pattern#modeless-dialogs

**Comment by zvonimirfras** [comment]
That link doesn't seem to be pointing to a specific place in the page. Should it be https://www.carbondesignsystem.com/patterns/dialog-pattern/#non-modal-dialogs instead or can we close this?

**Comment by cal-smith** [comment]
Yeah, that link is the correct one, I guess they changed the title since the issue was created
